docs/Using-Fleet/MDM-macOS-updates.md

```diff
@@ -71,7 +71,92 @@ Sometimes after the end user clicks "update" on the Nudge window, the end user's
 
 ## End user macOS update via built-in macOS notifications
 
-Built-in macOS update reminders are available for all Fleet instances. To trigger these reminders, run the ["Schedule an OS update" MDM command](https://developer.apple.com/documentation/devicemanagement/schedule_an_os_update).
+Built-in macOS update reminders are available in Fleet Free and Fleet Premium. 
+
+To trigger these reminders, we will do the following steps:
+
+1. Force a macOS update scan
+
+2. List available macOS updates
+
+3. Trigger macOS update reminder
+
+### Step 1: force a macOS update scan
+
+Use the request payload below when running a custom MDM command with Fleet. Documentation on how to run a custom command is [here](./MDM-commands#custom-commands).
+
+Request payload:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>Command</key>
+    <dict>
+        <key>ForceUpdateScan</key>
+        <true/>
+        <key>RequestType</key>
+        <string>ScheduleOSUpdateScan</string>
+    </dict>
+</dict>
+</plist>
+```
+
+### Step 2: list available macOS updates
```

Comment:
I know we kept them in your other PR but: think we could go without the ordered list for the steps in this section? Has the same indentation issue, but since these steps are so short and there's only 2, I feel like we could get rid of the numbering without sacrificing readability. (Alternatively, if you indent the content below the line that's numbered, it will all line up since there's only 1 code block.)

Since the formatting is the only issue & the content looks good, I'll go ahead and approve.

Reply:
Great suggestions Rachael. 

I tried to indent the content below the line but then decided to leave the formatting as-is for now. 

I think we can address the underlying indenting problem in an update to the markdown formatting. Filed an issue for this here: #12836